client_code/Tabulator/__init__.py

```diff
@@ -145,6 +145,9 @@ def set_filter(self, field, type=None, value=None):
         self._table.setFilter(field, type, value)
 
     def add_filter(self, field, type, value):
+        if callable(field):
+            filter_func = field
```

Comment:
Can we add the default args to type and value so that the type and value aren't required for a custom filter.